extensions/wikia/FlowTracking/FlowTracking.hooks.php

```diff
@@ -1,8 +1,24 @@
 <?php
 
 class FlowTrackingHooks {
+	const CREATE_PAGE_DIRECT_URL = 'create-page-direct-url';
+	const CREATE_PAGE_CONTRIBUTE_BUTTON = 'create-page-contribute-button';
+	const CREATE_PAGE_ARTICLE_REDLINK = 'create-page-article-redlink';
+	const CREATE_PAGE_SPECIAL_REDLINK = 'create-page-special-redlink';
 	const CREATE_PAGE_UNRECOGNIZED_FLOW = 'create-page-unrecognized';
 
+	public static function onMakeGlobalVariablesScript( &$vars ) {
+		$vars[ 'wgFlowTrackingFlows' ] = [
+			'CREATE_PAGE_DIRECT_URL' => static::CREATE_PAGE_DIRECT_URL,
+			'CREATE_PAGE_CONTRIBUTE_BUTTON' => static::CREATE_PAGE_CONTRIBUTE_BUTTON,
+			'CREATE_PAGE_ARTICLE_REDLINK' => static::CREATE_PAGE_ARTICLE_REDLINK,
+			'CREATE_PAGE_SPECIAL_REDLINK' => static::CREATE_PAGE_SPECIAL_REDLINK,
+			'CREATE_PAGE_UNRECOGNIZED_FLOW' => static::CREATE_PAGE_UNRECOGNIZED_FLOW
+		];
+
+		return true;
+	}
+
 	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$out, $skin". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)